### DISCUSSION TITLE
Not receiving 7.4.3 on Debian (Linux Mint LMDE)

### DISCUSSION THREAD

**Discussion by nicolasnobelis** [cyanfish/naps2#410] [Support]
Hello,

I was getting all Naps2 updates without any problem but I don't "see" the version 7.4.3.

I am running Linux Mint LMDE.

`/etc/apt/sources.list.d/naps2.list`:
deb [signed-by=/etc/apt/keyrings/naps2.gpg] https://downloads.naps2.com ./

`sudo apt-get update`
Hit :10 https://downloads.naps2.com ./ InRelease  

`apt-cache policy naps2 `
naps2:
  Installed: 7.4.2
  Candidate: 7.4.2
  Version table:
 *** 7.4.2 500
        500 https://downloads.naps2.com ./ Packages
        100 /var/lib/dpkg/status
     7.4.1 500
        500 https://downloads.naps2.com ./ Packages
     7.4.0 500
        500 https://downloads.naps2.com ./ Packages

Any idea which I don't see the 7.4.3 ?

**Comment by cyanfish** [comment]
Thanks for reporting this, it looks like there's some issue with my script to update the package metadata. It should be updated now.

**Reply by nicolasnobelis** [reply]
Yes it works now. Thanks :relieved:

**Comment by nicolasnobelis** [comment]
@cyanfish 8.1.0 is on the Debian repository but 8.1.1 is not.

Could it be this issue again ?

```
naps2:
  Installed: 8.1.0
  Candidate: 8.1.0
  Version table:
```

**Reply by cyanfish** [reply]
My bad, I was rushed to get the hotfix out and left the terminal without finishing the apt upload. Should be fixed now.

**Reply by nicolasnobelis** [reply]
I updated the repo, but still no 8.1.1 update.

And https://downloads.naps2.com/Packages also does not contain it...

**Reply by nicolasnobelis** [reply]
I just got the update, thanks !
